uamqp/_connection.py

```diff
@@ -475,6 +533,16 @@ def _process_outgoing_frame(self, channel, frame):
             raise ValueError("Connection not configured to allow pipeline send.")
         if self.state not in [ConnectionState.OPEN_PIPE, ConnectionState.OPEN_SENT, ConnectionState.OPENED]:
             raise ValueError("Connection not open.")
+        now = time.time()
+        if self._get_local_timeout(now) or self._get_remote_timeout(now):
+            self.close(
+                error=AMQPError(
+                    condition=ErrorCodes.InternalError.value,
```

Comment:
Why do we use `.value` here? We didn't on line 348?

Reply:
nice catch, it should not be using `.value` since extensible enum is used now.